multimedia/minisatip/files/minisatip.init

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
+start_service() {
+  procd_open_instance [minisatip]
+  procd_set_param command /usr/bin/minisatip -f -R /usr/share/minisatip/html
+
+  # respawn automatically if something died, be careful if you have an alternative process supervisor
+  # if process dies sooner than respawn_threshold, it is considered crashed and after 5 retries the service is stopped
+  procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
```

Comment:
Just `procd_set_param respawn` should be fine.  The local refs to `respawn_xxx` should be avoided.  It's bad initscripts controlled this way.